src/Stryker.Core/Stryker.Core/Instrumentation/ConditionalInstrumentationEngine.cs

```diff
@@ -22,15 +23,16 @@ public ParenthesizedExpressionSyntax PlaceWithConditionalExpression(ExpressionSy
                 SyntaxFactory.ConditionalExpression(
                     condition: condition,
                     whenTrue: mutated,
-                    whenFalse: original))
-            // Mark this node as a MutationConditional node. Store the MutantId in the annotation to retrace the mutant later
-            .WithAdditionalAnnotations(Marker);
+                    whenFalse: original)).
+        WithTriviaFrom(original).
```

Comment:
Does this always work? In some mutators, we mutate sub-nodes, so we might remove trivia there, but not re-add them here. Is that an issue?

Reply:
see previous comment: there shall be no sub node mutation. Otherwise it will not work; I failed to find a way to support both sub node mutations AND general trivia support.